app/src/main/java/randomappsinc/com/sqlpractice/Activities/WebActivity.java

```diff
@@ -49,7 +51,7 @@ public void onReceivedError(WebView view, WebResourceRequest req, WebResourceErr
                 onReceivedError(view, rerr.getErrorCode(), rerr.getDescription().toString(), req.getUrl().toString());
             }
         });
-
+        webView.getSettings().setCacheMode(WebSettings.LOAD_DEFAULT);
```

Comment:
I was about to write a very complicated mechanism to download the webpage's HTML into a file in internal storage. This one-liner seems a lot better, haha.